source/nodejs/adaptivecards-site/pages/_posts/2020/Community-Call-August.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 📣 Community Call - August, 2020 
+subtitle: v1.3 is released, and a real-world look at automating business processes with Cards and Power Automate
+date: 2020-08-24
+featured_image: https://developer.microsoft.com/en-us/office/blogs/wp-content/uploads/2020/08/A-screenshot-of-a-social-media-post-768x712.png
+github_username: matthidinger
+twitter: matthidinger
+---
+
+The Adaptive Cards community call for August, hosted by Matt Hidinger, also included guest speakers Hammad Hassan and Tomasz Poszytek.
+
+### Adaptive Cards v1.3 is now available!
+
+On this call we reviewed the [**Adaptive Cards v1.3**](https://github.com/microsoft/AdaptiveCards/releases/tag/2020.07) release, which includes the most-requested feature of the past 2 years: [**Input Validation and Input Labels**](https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/input-validation)!
+
+With this release you'll be able to mark which **Inputs** are required, as well as any regex-based pattern matching for text, and min/max ranges for things like numbers and dates.
+
+We've also streamlined the [Input schema](https://adaptivecards.io/explorer/Input.Text.html) by adding a **label** property, which improves the interaction model, automatically renders those red-asterisk "required field indicators", and greatly improves accessibility for Inputs.
+
+**Screenshot of an Input with a *label* and *isRequired* validation:**
+
+➡ [Try it yourself in the Designer](https://adaptivecards.io/designer/index.html?card=/payloads/InputsWithValidation.template.json&data=/payloads/InputsWithValidation.data.json)
+
+![Screenshot of input validation in a card](https://developer.microsoft.com/en-us/office/blogs/wp-content/uploads/2020/08/Designer.png)
+
+**Example Card with Input Validation and Labels:**
+
+```json
+{
+    "type": "AdaptiveCard",
```

Comment:
these spaces are showing as red in the preview for both August and July posts... looks fine in September post. not sure if that'll actually show on the site though...